internal/cache/cache.go

```diff
@@ -0,0 +1,238 @@
+/*
+Original work Copyright (c) 2012-2019 Patrick Mylund Nielsen and the go-cache contributors
```

Comment:
We need both licences in here, please see https://github.com/fluxcd/pkg/blob/main/ssh/knownhosts/knownhosts.go

Reply:
Let's give it another try.

Reply:
Looks Ok now!